recipes/gnu-getopt/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+export CFLAGS="-I${PREFIX}/include"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib -L${PREFIX}/include -lintl"
+
+cd "$SRC_DIR"
```

Comment:
This is already done by `conda-build`. So can be dropped.

Reply:
Right, thanks.